go/vt/vtctl/vtctl.go

```diff
@@ -3328,6 +3328,14 @@ func commandOnlineDDL(ctx context.Context, wr *wrangler.Wrangler, subFlags *flag
 			uuid = arg
 			query, bindErr = sqlparser.ParseAndBind(`update _vt.schema_migrations set migration_status='retry' where migration_uuid=%a`, sqltypes.StringBindVariable(arg))
 		}
+	case "complete":
+		{
+			if arg == "" {
+				return fmt.Errorf("UUID required")
+			}
+			uuid = arg
+			query, bindErr = sqlparser.ParseAndBind(`update _vt.schema_migrations set migration_status='complete' where migration_uuid=%a`, sqltypes.StringBindVariable(arg))
+		}
```

Comment:
I haven't seen this style before. Not objecting to it, but the curly braces around the code block aren't really required, right?

Reply:
Right -- these are not needed. To me it reads clearer with the curly braces, though I realize the convention is to not use them. I've just removed them and committed the braceless case statement.